components/Header.js

```diff
@@ -19,3 +32,10 @@ const StyledHeader = styled.header`
   box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
   position: relative;
 `;
+
+const Headline = styled.h2`
+  font-size: 18px;
```

Comment:
I recommend to not work with `px` for `font-size` for accessibility reasons. [See this article for more information.](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#accessibility-considerations-5)